android/app/src/main/java/org/mozilla/magnet/permissions/CheckGooglePlayServices.java

```diff
@@ -0,0 +1,54 @@
+package org.mozilla.magnet.permissions;
+
+import android.app.Activity;
+
+import com.google.android.gms.common.ConnectionResult;
+import com.google.android.gms.common.GoogleApiAvailability;
+
+/**
+ * Created by wilsonpage on 09/11/2016.
+ */
+class CheckGooglePlayServices extends Check {
```

Comment:
I don't think I want to make this `public` as it means the class is reachable outside of the `permissions` package which isn't the intention.